<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImgurOptions</class>
 <widget class="QWidget" name="ImgurOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="userLabel">
     <property name="text">
      <string>User:</string>
     </property>
     <property name="buddy">
      <cstring>authButton</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="authUserLabel">
     <property name="text">
      <string>&lt;i&gt;none</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="authButton">
     <property name="text">
      <string>Authorize</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="albumLabel">
     <property name="text">
      <string>Upload to album:</string>
     </property>
     <property name="buddy">
      <cstring>albumComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="albumComboBox">
     <item>
      <property name="text">
       <string>- None -</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="refreshAlbumButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="helpLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Note&lt;/span&gt;: Clicking on the &amp;quot;Authorize&amp;quot; button will open the Imgur website, where you will need to log in with the account you wish to use. They will provide an authentication PIN that you must input into Lightscreen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
